api-proposal.md

```diff
@@ -0,0 +1,77 @@
+# Proposal for ecosystem website api.
+
+## API's
+
+---
+
+### Package List
+
+these 3 return a list of packages, they would all use the same schema.
+
+- GET `/api/v1/packages`
+- GET `/api/v1/categories/:category`
+- GET `/api/v1/search?query=:query`
+  ```
+    items: [<package|quick>]
+  ```
+
+### Package
+
+- GET `/api/v1/package/:package`
+  ```
+  data: <package>
+  ```
+- POST `/api/v1/package`
+  ```
+  data: {
+    package: <package>
+    releases: [<release>]
```

Comment:
This would probably be an array of `id`'s.

Reply:
I agree.